src/install/kubernetes/manifest/install-manifest.mdx

```diff
@@ -0,0 +1,98 @@
+        namespace: newrelic
+    newrelic-infrastructure.privileged:
+        enabled: false
+    global.lowDataMode:
+        enabled: true
+    kube-state-metrics.image.tag:
+        "'${KSM_IMAGE_VERSION}'"
+    kube-state-metrics:
+        # Deploy kube-state-metrics in the cluster.
+        # Set this to true unless it is already deployed.
+        enabled: true
+    kubeEvents:
+        # Report Kubernetes events
+        enabled: true
+    newrelic-prometheus-agent:
+        # Automatically scrape prometheus metrics for annotated services in the cluster
+        enabled: true
+    newrelic-prometheus-agent.lowDataMode:
+        enabled: true
+    newrelic-prometheus-agent.config.kubernetes.integrations_filter:
+        enabled: false
+    logging.enabled:
+        enabled: true
+    newrelic-logging.lowDataMode:
+        # Report logs for containers running in the cluster
+        enabled: true
+    newrelic-logging.fluentBit.linuxMountPath:
+        /var/log
+    storageClass:
+        enabled: standard-rwx
+    ```
+
+2. Run this command to apply the manifest generated:
+
+    ```shell
+    kubectl -n newrelic apply -f newrelic-manifest.yaml
+    ```
+
+3.  If you're going to use a GKE AutoPilot cluster, your Helm command should include the following:
+
+    ```yaml
+    --set newrelic-infrastructure.controlPlane.enabled=false \
+    --set newrelic-infrastructure.kubelet.config.scheme=http \
+    --set newrelic-infrastructure.kubelet.config.port=10255 \
+    ```
+
+    [Fluent Bit](https://fluentbit.io/) can use a FileStore volume to prevent data loss or duplicated logs during Fluent Bit pod restarts or redeploys.  Google FileStore API needs to be enabled in your GKE project. Check this [document](https://support.google.com/googleapi/answer/6158841) to know more about enabling APIs. 
+
+    To use FileStore, add the following:
+
+    ```yaml
+    --set newrelic-logging.fluentBit.persistence.mode=persistentVolume
+    --set newrelic-logging.fluentBit.persistence.persistentVolume.storageClass=standard-rwx
+    ```
+
+    <Callout variant="important">
+    The use of FileStore will result in additional costs, as charged by Google. See your Google Cloud Admin or the [FileStore](https://cloud.google.com/filestore/pricing) doc for more info.
+    </Callout>
+
+    If you don't want to use FileStore:
+
+    * Set `"newrelic-logging.fluentBit.persistence.mode"` to `"none"`
+    * Remove `"newrelic-logging.fluentBit.persistence.persistentVolume.storageClass"`
+
+4. Decide if you want to install and configure the Prometheus endpoints to scrape. See [Install and configure the Prometheus agent on Kubernetes clusters](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/) for more info. We recommend to scrape all Prometheus endpoints except core Kubernetes system metrics. Kubernetes integration already collects core Kubernetes system metrics in other components. 
```

Comment:
```suggestion
4. Decide if you want to install and configure the Prometheus endpoints to scrape. We recommend to scrape all Prometheus endpoints except core Kubernetes system metrics. Kubernetes integration already collects core Kubernetes system metrics in other components. See [Install and configure the Prometheus agent on Kubernetes clusters](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/) for more info. 
```

Reply:
yes, much better. Thank you, @homelessbirds